UserTools/RunControl/RunControl.cpp

```diff
@@ -15,14 +15,19 @@ RunControl::RunControl():Tool(){}
 bool RunControl::Initialise(std::string configfile, DataModel &data){
   
   InitialiseTool(data);
-  InitialiseConfiguration(configfile);
-  //m_variables.Print();
   
-  // printf("d1\n");
+  m_configfile = configfile;
+  InitialiseConfiguration(m_configfile);
+  ExportConfiguration();
+  
+  Log(m_tool_name+" initialised configuration",v_debug,m_verbose);
+  std::cout<<m_tool_name<<" m_variables:\n--------------\n";
+  m_variables.Print();
```

Comment:
again debug statments need to be removed

Reply:
what do you mean again, this is the first instance. 
But these were left in as these changes are not yet proven, needs testing.